### DISCUSSION TITLE
prisma.$queryRaw<SomeType[]> is not converting Date column values

### DISCUSSION THREAD

**Discussion by thesobercoder** [prisma/prisma#5915] [Q&A] [answered]
If I have a type like

```ts
export type SomeType {
    col: Date
}
```

and I try to run a raw query with this object

```ts
const res = prisma.$queryRaw<SomeType[]>(`EXEC usp_SomeSproc`);
```

the resulting list of objects is being returned with the date column as a string and not as a JavaScript date.

Am I doing something wrong here?

**Comment by ryands17** [comment, marked as answer]
@thesobercoder 👋 

I would suggest opening an issue for this [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=bug_report.md&title=) so that we can look into this :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
